### PR TITLE
Issue#16

### PR DESCRIPTION
**Pull request by mahdihaghverdi**
What's Changed:
fixed Issue #16 

### Changed
- `unwind` is now an instance method, not a `staticmethod`

### CODE REVIEW

File: aggify/aggify.py
[the remark this answers is not in the document]
Comment (mahdihaghverdi) [reply aggify/aggify.py:1]: Oh, Thanks

File: aggify/aggify.py [hunk 1]
```diff
@@ -203,17 +203,15 @@ def __to_aggregate(self, query: dict[str, Any]) -> None:
...
-                    ]
+                self.pipelines.append(
+                    self.__lookup(
+                        from_collection=from_collection._meta["collection"],  # noqa
+                        local_field=local_field,
+                        as_name=as_name,
+                    )
                 )
                 self.unwind(as_name)
+                self.pipelines.extend([{"$match": match} for match in matches])
```
Comment (seyed-dev) [inline aggify/aggify.py:214]: If only one thing is going to be added to the list, use append

Comment (mahdihaghverdi) [reply aggify/aggify.py:214]: oh, I thought I've used extend, but it seems my fingers have written append 😂😂

Thanks 😁

Comment (mahdihaghverdi) [reply aggify/aggify.py:214]: there may be more than one `$match` stage, so extending is necessary

**Pull request merged**